compiler/src/dmd/mars.d

```diff
@@ -343,35 +343,40 @@ const(char)[] parse_conf_arg(Strings* args)
  * override any value.
  * Note that if `-defaultlib=` or `-debuglib=` was used,
  * we don't override that either.
+ * Params:
+ *	target = parameters set by user
+ *	defaultlibname = set based on `target`
+ *	debuglibname = set based on `target`
  */
-void setDefaultLibrary(ref Param params, const ref Target target)
+pure @safe
+void setDefaultLibraries(const ref Target target, ref const(char)[] defaultlibname, ref const(char)[] debuglibname)
 {
-    if (driverParams.defaultlibname is null)
+    if (defaultlibname is null)
     {
         if (target.os == Target.OS.Windows)
         {
-            driverParams.defaultlibname = target.isX86_64 ? "phobos64" : "phobos32mscoff";
+            defaultlibname = target.isX86_64 ? "phobos64" : "phobos32mscoff";
```

Comment:
at some point I'd love to rename `phobos32mscoff` to `phobos32`, but I could never figure out the correct build system invocation corrections.

Reply:
Add a bugzilla issue for that!

Reply:
Done: https://issues.dlang.org/show_bug.cgi?id=24576